rasa/nlu/featurizers/sparse_featurizer/regex_featurizer.py

```diff
@@ -45,58 +52,60 @@ def train(
         self._add_lookup_table_regexes(training_data.lookup_tables)
 
         for example in training_data.training_examples:
-            updated = self._text_features_with_regex(example)
-            example.set(MESSAGE_VECTOR_FEATURE_NAMES[MESSAGE_TEXT_ATTRIBUTE], updated)
+            self._text_features_with_regex(example)
 
     def process(self, message: Message, **kwargs: Any) -> None:
+        self._text_features_with_regex(message)
 
-        updated = self._text_features_with_regex(message)
-        message.set(MESSAGE_VECTOR_FEATURE_NAMES[MESSAGE_TEXT_ATTRIBUTE], updated)
-
-    def _text_features_with_regex(self, message):
+    def _text_features_with_regex(self, message: Message) -> None:
         if self.known_patterns:
-            extras = self.features_for_patterns(message)
-            return self._combine_with_existing_features(message, extras)
-        else:
-            return message.get(MESSAGE_VECTOR_FEATURE_NAMES[MESSAGE_TEXT_ATTRIBUTE])
+            for attribute in [MESSAGE_TEXT_ATTRIBUTE, MESSAGE_RESPONSE_ATTRIBUTE]:
+                extras = self._features_for_patterns(message, attribute)
```

Comment:
what will happen during prediction time here for response features, because response doesn't exist during prediction